src/main/java/fr/esgi/al5_2/Tayarim/mappers/AdministrateurMapper.java

```diff
@@ -0,0 +1,48 @@
+package fr.esgi.al5_2.Tayarim.mappers;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.AdministrateurCreationDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.AdministrateurDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireCreationDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireDTO;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import lombok.NonNull;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.time.LocalDateTime' import. Should be before 'org.springframework.stereotype.Component'.